src/rpc/mining.cpp

```diff
@@ -885,12 +886,13 @@ UniValue getblocksubsidy(const UniValue& params, bool fHelp)
             "      \"recipient\" : \"...\",      (string) A description of the funding stream recipient.\n"
             "      \"specification\" : \"url\",  (string) A URL for the specification of this funding stream.\n"
             "      \"value\" : x.xxx           (numeric) The funding stream amount in " + CURRENCY_UNIT + ".\n"
+            "      \"[addresses]\" :           (array) An array of addresses for the funding stream recipient.\n"
```

Comment:
An array of addresses doesn't make sense for this API. `getblocksubsidy` returns the block subsidy (in its various splits) as of a specific height. A given funding stream may have multiple addresses across multiple blocks, but can only be directed to a single address within a single block.

The intent of #4728 is that a miner that is constructing a block for height `X` can call `getblocksubsidy X` to know what funding stream outputs they need to include for that height (which is sufficient as funding streams are defined relative to heights, not specific chains). I would similarly suggest that we add `valueZat` to each recipient as well, for removal of ambiguity.